omod/src/main/java/org/openmrs/module/htmlformentryui/fragment/controller/htmlform/EnterHtmlFormFragmentController.java

```diff
@@ -327,11 +346,15 @@ private void keepTimeComponentOfEncounterIfDateComponentHasNotChanged(Date previ
 	}
 	
 	private void setupModel(FragmentModel model, FormEntrySession fes, VisitDomainWrapper visitDomainWrapper,
-	        Boolean createVisit) {
+	        Boolean createVisit, String visitStartDatetime, String visitStopDatetime, UiUtils ui) {
 		
-		model.addAttribute("currentDate", (new DateMidnight()).toDate());
+		String currentDate = ui.convertTimezones() ? ui.format(new Date()) : new Date().toString();
```

Comment:
Of what format will `Date().toString` be?

Reply:
I wil change that to be `ui.dateToISOString(new Date())` so its consistent with others dates above.